packages/vm/src/evm/evm.ts

```diff
@@ -646,6 +644,68 @@ export default class EVM {
     }
   }
 
+  /**
+   * @ignore
+   */
+  async runCall(opts: RunCallOpts): Promise<EVMResult> {
+    const block = opts.block ?? Block.fromBlockData({}, { common: this._common })
+    this._block = block
+
+    const txContext = new TxContext(
+      opts.gasPrice ?? BigInt(0),
+      opts.origin ?? opts.caller ?? Address.zero()
+    )
+    this._tx = txContext
+
+    const message = new Message({
+      caller: opts.caller,
+      gasLimit: opts.gasLimit ?? 0xffffffn,
+      to: opts.to ?? undefined,
+      value: opts.value,
+      data: opts.data,
+      code: opts.code,
+      depth: opts.depth ?? 0,
+      isCompiled: opts.isCompiled ?? false,
+      isStatic: opts.isStatic ?? false,
+      salt: opts.salt ?? null,
+      selfdestruct: opts.selfdestruct ?? {},
+      delegatecall: opts.delegatecall ?? false,
+    })
+
+    return this.executeMessage(message)
+  }
+
+  /**
+   * Bound to the global VM and therefore
+   * shouldn't be used directly from the evm class
+   * @hidden
+   */
```

Comment:
This method should not be hidden since it will become part of the public API and rather get a proper code doc introduction.

Reply:
Removed hidden.